porting justin's edits into GH (#253)
jaybee84/ml-in-rd: content/03.combining-datasets.md

## Constructing machine learning-ready rare disease datasets

> High-throughput ‘omic’ assays can generate thousands to billions of measurements from whole transcriptome and whole genome sequencing respectively, resulting in high-dimensional dataset.

Reviewer: ```suggestion
High-throughput ‘omic’ assays can generate thousands to billions of measurements from whole transcriptome and whole genome sequencing respectively, resulting in high-dimensional datasets.
```

> A typical rare disease dataset consists of a small number of samples[@doi:10.1186/s13023-020-01424-6] leading to the “curse of dimensionality” in which the feature space is much larger than the sample space, can increase the difficulty in building highly generalizable models [@doi:10.1038/nrc2294].

Reviewer: ```suggestion
A typical rare disease dataset consists of a small number of samples[@doi:10.1186/s13023-020-01424-6] leading to the “curse of dimensionality” in which the feature space is much larger than the sample space, increasing the difficulty in building highly generalizable models [@doi:10.1038/nrc2294].
```

A larger feature space can contribute to higher data missingness (_sparsity_), more dissimilarity between samples (_variance_), and increased redundancy among individual features or combinations (_multicollinearity_) [@doi:10.1038/s41592-018-0019-x], all of which contribute to challenges in ML implementation.

An important factor in ML is model performance: the accuracy of a supervised model in identifying patterns relevant for a biological question, or the reliability of an unsupervised model in identifying hypothetical biological patterns supported by post-hoc validation.
When small sample sizes compromise an ML model’s performance, two approaches can be taken to manage sparsity, variance, and multicollinearity: 1) increase the number of samples, 2) improve the quality of samples.
In the first approach, appropriate training, evaluation, and held-out validation sets could be constructed by combining multiple rare disease cohorts (Figure [@fig:1]a, Box 2).
When combining datasets, special attention should be directed towards data harmonization since data collection methods can differ between cohorts.

> Without careful selection of aggregation methods, one may introduce technical – in contrast to biological variability into the combined dataset and negatively impact the ML model’s ability to learn or detect meaningful signals.

Reviewer 2: Think this needs a closing em dash
```suggestion
Without careful selection of aggregation methods, one may introduce technical – in contrast to biological – variability into the combined dataset and negatively impact the ML model’s ability to learn or detect meaningful signals.
```
